From c46b7c6e0662ba9a8fe0809da2f095075a0d7152 Mon Sep 17 00:00:00 2001
From: "winton.liu" <admin@brahmaos.io>
Date: Thu, 29 Nov 2018 17:24:19 +0800
Subject: [PATCH] build: add BRAHMAOS_VERSION variable

add BRAHMAOS_VERSION and set property ro.build.version.brahmaos

Change-Id: I4d4ff4ee31242efbed7391191f5290f096ae6a5b
Signed-off-by: winton.liu <admin@brahmaos.io>
---
 core/build_id.mk   | 1 +
 core/dumpvar.mk    | 1 +
 tools/buildinfo.sh | 1 +
 3 files changed, 3 insertions(+)

diff --git a/core/build_id.mk b/core/build_id.mk
index 86ab190..e17bf69 100644
--- a/core/build_id.mk
+++ b/core/build_id.mk
@@ -19,3 +19,4 @@
 # capitalized by convention.
 
 export BUILD_ID=BrahmaOS_0.1.2
+export BRAHMAOS_VERSION=0.1.2
diff --git a/core/dumpvar.mk b/core/dumpvar.mk
index acae48e..2ece1b3 100644
--- a/core/dumpvar.mk
+++ b/core/dumpvar.mk
@@ -3,6 +3,7 @@
 print_build_config_vars := \
   PLATFORM_VERSION_CODENAME \
   PLATFORM_VERSION \
+  BRAHMAOS_VERSION \
   TARGET_PRODUCT \
   TARGET_BUILD_VARIANT \
   TARGET_BUILD_TYPE \
diff --git a/tools/buildinfo.sh b/tools/buildinfo.sh
index d214aab..eb1a076 100755
--- a/tools/buildinfo.sh
+++ b/tools/buildinfo.sh
@@ -6,6 +6,7 @@ echo "# autogenerated by buildinfo.sh"
 echo "ro.build.id=$BUILD_ID"
 echo "ro.build.display.id=$BUILD_DISPLAY_ID"
 echo "ro.build.version.incremental=$BUILD_NUMBER"
+echo "ro.build.version.brahmaos=$BRAHMAOS_VERSION"
 echo "ro.build.version.sdk=$PLATFORM_SDK_VERSION"
 echo "ro.build.version.preview_sdk=$PLATFORM_PREVIEW_SDK_VERSION"
 echo "ro.build.version.codename=$PLATFORM_VERSION_CODENAME"
-- 
2.7.4

